pinecone/data/index.py

```diff
@@ -69,15 +69,25 @@ class Index():
     For improved performance, use the Pinecone GRPC index client.
     """
 
-    def __init__(self, api_key: str, host: str, pool_threads=1, **kwargs):
-        api_key = api_key or kwargs.get("api_key", None)
-        host = host or kwargs.get('host', None)
-        pool_threads = pool_threads or kwargs.get("pool_threads")
```

Comment:
Were these removed because we don't necessarily need to check for them in `kwargs` anymore?

Reply:
I realized that the code wasn't doing anything. If you have a function that takes `**kwargs`, only unnamed keyword args that are not explicitly listed are present in the kwargs dictionary.

```python
>>> def see_kwargs(api_key, **kwargs):
...   print(kwargs)
...
>>> see_kwargs(api_key='xxx', extra_stuff=True)
{'extra_stuff': True}
>>> see_kwargs('xxx', more_extra=True)
{'more_extra': True}
```